src/main/java/org/folio/inventory/dataimport/handlers/actions/DeleteAuthorityEventHandler.java

```diff
@@ -0,0 +1,67 @@
+package org.folio.inventory.dataimport.handlers.actions;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import org.folio.ActionProfile;
+import org.folio.Authority;
+import org.folio.DataImportEventPayload;
+import org.folio.inventory.dataimport.cache.MappingMetadataCache;
+import org.folio.inventory.dataimport.exceptions.DataImportException;
+import org.folio.inventory.domain.AuthorityRecordCollection;
+import org.folio.inventory.storage.Storage;
+import org.folio.rest.jaxrs.model.ProfileSnapshotWrapper;
+
+import static java.lang.String.format;
+import static org.folio.ActionProfile.FolioRecord.AUTHORITY;
+import static org.folio.ActionProfile.FolioRecord.MARC_AUTHORITY;
+import static org.folio.ActionProfile.Action.DELETE;
+import static org.folio.DataImportEventTypes.DI_SRS_MARC_AUTHORITY_RECORD_DELETED;
+import static org.folio.rest.jaxrs.model.ProfileSnapshotWrapper.ContentType.MAPPING_PROFILE;
+
+public class DeleteAuthorityEventHandler extends AbstractAuthorityEventHandler {
+  protected static final String ERROR_DELETING_AUTHORITY_MSG_TEMPLATE = "Failed deleting Authority. Cause: %s, status: '%s'";

```

Comment:
There will be no mapping for delete action as you can see in [MODDICONV-228](https://issues.folio.org/browse/MODDICONV-228). You should expect authority ID in the payload.